src/lib.rs

```diff
@@ -137,6 +137,9 @@ extern crate sha1;
 #[cfg_attr(test, macro_use)]
 extern crate slog;
 
+mod builder;
+pub use builder::UuidBuilder;
```

Comment:
make the `builder` module `pub` and instead of the reexport here, reexport in `prelude` module